### ISSUE TITLE
Cannot access video stream

### ISSUE THREAD

**Issue by Arturokin12** [schmich/instascan#107]
Hello, i've been getting this error, i have my website on GoDaddy, and i've been using instascan for months, but now i'm getting this error on every browser, hope you guys can help me

![image](https://user-images.githubusercontent.com/20196515/33247404-a2a57412-d2fc-11e7-889f-ca61df75356d.png)

**Comment by mairh** [comment]
Is your website hosted over https or http?

**Comment by Arturokin12** [comment]
yes, i was doing research and, chrome doesn't allow this 'webcam' use in
the last update, what a shame, that's now the problem, thanks for answering

2017-11-27 9:41 GMT-03:00 Ujjwal Mairh <notifications@github.com>:

**Comment by leplatrem** [comment]
See #87 

(and also check out Let's Encrypt which seems to be manageable on GoDaddy)

**Comment by meticulousCraftman** [comment]
I encountered the same error.
Android - 7.0 (Nougat)
Chrome Version - 73.0.3683.90

`
Error: Cannot access video stream (NotReadableError)
`

Is there any other QR code scanning library?

**Comment by lewis-machilika** [comment]
many websites are not supporting instascan
